plugins/websocket/servers/csharp/Binaries/.gitattributes

```diff
@@ -0,0 +1,4 @@
+Neurala.VIA.WebSocket.dll filter=lfs diff=lfs merge=lfs -text
+Newtonsoft.Json.dll filter=lfs diff=lfs merge=lfs -text
+Test.exe filter=lfs diff=lfs merge=lfs -text
+websocket-sharp.dll filter=lfs diff=lfs merge=lfs -text
```

Comment:
@Deneuki can we more that to top level, and mark any dll? Should be something like: `*.dll filter=lfs diff=lfs merge=lfs -text`